cabal.project

```diff
@@ -51,3 +51,41 @@ packages:
 tests: True
 optimization: False
 -- reorder-goals: True
+
+allow-newer: all
+
+-- https://github.com/GaloisInc/cereal/pull/103
+source-repository-package
+    type: git
+    location: https://github.com/parsonsmatt/cereal
+    tag: b4acf0b778a1d5da305f4de013ce7f32b53d8282
+
+-- https://github.com/haskell-foundation/foundation/pull/564
+source-repository-package
+    type: git
+    location: https://github.com/parsonsmatt/foundation
+    tag: 688c32ccd9a951bc96dd09423a6e6684f091d510
+    subdir: basement
+    subdir: foundation
+
+-- https://github.com/vincenthz/hs-memory/pull/93
+source-repository-package
+    type: git
+    location: https://github.com/parsonsmatt/hs-memory
+    tag: 296b79424854eae293f6ba09b5308a0bf4dfd6d5
+
+
+-- https://github.com/snap-framework/io-streams/pull/83
```

Comment:
Link is wrong, should be https://github.com/snapframework/io-streams/pull/83

Reply:
Cheers that's been merged now so I've removed the line